.github/workflows/update-deployment.yml

```diff
@@ -10,8 +10,6 @@ on:
         default: 'dev'
```

Comment:
Default should get changed to 'sandbox' I imagine.

Reply:
Wouldn't have spotted that, but I agree.